action.yml

```
@@ -26,6 +29,14 @@ inputs:
     description: 'CloudFoundry Organization to target'
     required: true
 
+  space-suffix:
+    description: >
+      The suffix to append to the targeted space. If the targeted space is "dev"
+      and the space-suffix is "-public" the resulting "dev-public" space will be
```

```suggestion
      and the space-suffix is "public" the resulting "dev-public" space will be
```
Dash is added by action

---

Opted to remove default delimiter enforcement by the action. Choice of delimiter should be left to the developer and may not always be what we force here, so removing it allows arbitrary delimiters in the parameter.